src/components/message_list_view/message_row/base.rs

```diff
@@ -53,17 +53,17 @@ mod imp {
         }
 
         fn show_message_menu(&self, x: i32, y: i32) {
-            let obj = self.obj();
-            let chat_history = obj.ancestor(ChatHistory::static_type()).unwrap();
-            let menu = chat_history
-                .downcast_ref::<ChatHistory>()
-                .unwrap()
-                .message_menu();
-
-            menu.set_pointing_to(Some(&gdk::Rectangle::new(x, y, 0, 0)));
-            menu.unparent();
-            menu.set_parent(&*obj);
-            menu.popup();
+            // let obj = self.obj();
+            // let chat_history = obj.ancestor(ChatHistory::static_type()).unwrap();
+            // let menu = chat_history
+            //     .downcast_ref::<ChatHistory>()
+            //     .unwrap()
+            //     .message_menu();
+
+            // menu.set_pointing_to(Some(&gdk::Rectangle::new(x, y, 0, 0)));
+            // menu.unparent();
+            // menu.set_parent(&*obj);
+            // menu.popup();
         }
```

Comment:
We need to handle message menus in a way that's not dependent on ChatHistory anymore.